modules/identityLinkIdSystem.js

```diff
@@ -39,8 +44,13 @@ export const identityLinkSubmodule = {
     }
     const hasGdpr = (consentData && typeof consentData.gdprApplies === 'boolean' && consentData.gdprApplies) ? 1 : 0;
     const gdprConsentString = hasGdpr ? consentData.consentString : '';
+    const tcfPolicyV2 = !!((consentData && consentData.vendorData && consentData.vendorData.tcfPolicyVersion && consentData.vendorData.tcfPolicyVersion === 2));
```

Comment:
could you use a `deepEquals()` here instead?

Reply:
Hello,
did you maybe thought about `utils.deepAccess` instead of `utils.deepEqual` ?

Reply:
Ah yes, sorry, I did mean `deepAccess`:

```javascript
const tcfPolicyV2 = !!((utils.deepAccess(consentData, 'vendorData.tcfPolicyVersion') === 2))
```

Reply:
No problem. We changed that to:
`const tcfPolicyV2 = utils.deepAccess(consentData, 'vendorData.tcfPolicyVersion') === 2;`
We pushed those changes.
If that is okay?

Reply:
yes, that should be fine. @mike-chowla will do the full review of the rest of the code, I just wanted to point out this optimization